scripts/build-image.sh

```diff
@@ -0,0 +1,84 @@
+#!/usr/bin/env sh
+
+# Where's my apt?
+APTGET="$( which apt-get )"
+
+# I'm not here, i won't interact with you
+APTGETCMD="env DEBIAN_FRONTEND=noninteractive ${APTGET}"
+
+# Let's tell apt to not install recommendations,
+# assume always a positive answer, and allow unauthenticated repos
+# Dpkg: you need to install configurations always. Ah, and be quiet.
+APTGETOPTS="-qq \
+            -o Apt::Install-Recommends=false \
+            -o Apt::Get::Assume-Yes=true \
+            -o Apt::Get::AllowUnauthenticated=true \
+            -o DPkg::Options::=--force-confmiss \
+            -o DPkg::Options::=--force-confnew \
+            -o DPkg::Options::=--force-overwrite \
+            -o DPkg::Options::=--force-unsafe-io"
+
+# List of software we need
+DPKGDEPENDS="bzr \
```

Comment:
What about use a new `requirements_apt.txt` file like as `requirements.txt` for pip?